src/MPL-2.0.xml

```diff
@@ -0,0 +1,436 @@
+<spdx name="Mozilla Public License 2.0" identifier="MPL-2.0" osi-approved="true">
+  <urls>
+    <url>http://www.mozilla.org/MPL/2.0/</url>
+    <url>http://opensource.org/licenses/MPL-2.0</url>
+  </urls>
+  <notes>This license was released in January 2012. This license list entry is for use when the standard MPL 2.0 is
+         used, as indicated by the standard header (Exhibit A but no Exhibit B).</notes>
+  <license>
+    <title>
+      <p>Mozilla Public License Version 2.0</p>
+    </title>
+    <body>
+      <list>
+        <li>
+          <b>1.</b>
+          <p>Definitions</p>
+        </li>
+        <list>
```

Comment:
This list should be part of the previous list. Other lists elsewhere in the document need similar attention.